_posts/2019-06-19-load-image-data.md

```diff
@@ -0,0 +1,1095 @@
+  <line x1="17" y1="7" x2="17" y2="68" />
+  <line x1="24" y1="14" x2="24" y2="75" />
+  <line x1="31" y1="21" x2="31" y2="82" />
+  <line x1="38" y1="28" x2="38" y2="89" />
+  <line x1="45" y1="35" x2="45" y2="96" />
+  <line x1="52" y1="42" x2="52" y2="103" />
+  <line x1="59" y1="49" x2="59" y2="110" />
+  <line x1="66" y1="56" x2="66" y2="117" />
+  <line x1="73" y1="63" x2="73" y2="124" />
+  <line x1="80" y1="70" x2="80" y2="131" style="stroke-width:2" />
+
+  <!-- Colored Rectangle -->
+  <polygon points="10.000000,0.000000 80.588235,70.588235 80.588235,131.722564 10.000000,61.134328" style="fill:#ECB172A0;stroke-width:0"/>
+
+  <!-- Horizontal lines -->
+  <line x1="10" y1="0" x2="55" y2="0" style="stroke-width:2" />
+  <line x1="17" y1="7" x2="62" y2="7" />
+  <line x1="24" y1="14" x2="69" y2="14" />
+  <line x1="31" y1="21" x2="77" y2="21" />
+  <line x1="38" y1="28" x2="84" y2="28" />
+  <line x1="45" y1="35" x2="91" y2="35" />
+  <line x1="52" y1="42" x2="98" y2="42" />
+  <line x1="59" y1="49" x2="105" y2="49" />
+  <line x1="66" y1="56" x2="112" y2="56" />
+  <line x1="73" y1="63" x2="119" y2="63" />
+  <line x1="80" y1="70" x2="126" y2="70" style="stroke-width:2" />
+
+  <!-- Vertical lines -->
+  <line x1="10" y1="0" x2="80" y2="70" style="stroke-width:2" />
+  <line x1="55" y1="0" x2="126" y2="70" style="stroke-width:2" />
+
+  <!-- Colored Rectangle -->
+  <polygon points="10.000000,0.000000 55.850746,0.000000 126.438982,70.588235 80.588235,70.588235" style="fill:#ECB172A0;stroke-width:0"/>
+
+  <!-- Horizontal lines -->
+  <line x1="80" y1="70" x2="126" y2="70" style="stroke-width:2" />
+  <line x1="80" y1="131" x2="126" y2="131" style="stroke-width:2" />
+
+  <!-- Vertical lines -->
+  <line x1="80" y1="70" x2="80" y2="131" style="stroke-width:2" />
+  <line x1="126" y1="70" x2="126" y2="131" style="stroke-width:2" />
+
+  <!-- Colored Rectangle -->
+  <polygon points="80.588235,70.588235 126.438982,70.588235 126.438982,131.722564 80.588235,131.722564" style="fill:#ECB172A0;stroke-width:0"/>
+
+  <!-- Text -->
+  <text x="103.513608" y="151.722564" font-size="1.0rem" font-weight="100" text-anchor="middle" >768</text>
+  <text x="146.438982" y="101.155399" font-size="1.0rem" font-weight="100" text-anchor="middle" transform="rotate(-90,146.438982,101.155399)">1024</text>
+  <text x="35.294118" y="116.428446" font-size="1.0rem" font-weight="100" text-anchor="middle" transform="rotate(45,35.294118,116.428446)">2010</text>
+</svg>
+</td>
+</tr>
+</table>
+
+
+on which some day we'll eventually be able to perform complex calculations.
+
+<img src="https://raw.githubusercontent.com/mrocklin/raw-host/gh-pages/images/aollsm-index-1.jpg"
+     width="45%"
+     alt="Light Microscopy data rendered with NVidia IndeX">
+<img src="https://raw.githubusercontent.com/mrocklin/raw-host/gh-pages/images/aollsm-index-2.jpg"
+     width="45%"
+     alt="Light Microscopy data rendered with NVidia IndeX">
```

Comment:
Text that describes what kind of data it is, e.g. light microscopy of ..., and what we would like to learn from it could provide context and motivation for a reader.

Reply:
Imaging the organelles of precursor neuronal cells in the hind brain of a developing zebrafish embryo using Lattice light sheet microscopy data with adaptive optics. 
Data source: 
Figure 3; Movie 4
https://science.sciencemag.org/content/360/6386/eaaq1392
pre-print for folks that can't access science journal
https://www.biorxiv.org/content/10.1101/243352v2

Reply:
@rxist525 awesome!

Reply:
I've added links to the paper and preprint, but also added a disclaimer that this workload should be generally applicable.